src/main/java/build/buildfarm/worker/MatchStage.java

```diff
@@ -33,9 +33,11 @@ protected void iterate() throws InterruptedException {
     if (!output.claim()) {
       return;
     }
-    worker.instance.match(worker.config.getPlatform(), worker.config.getRequeueOnFailure(), (operation) -> {
-      return fetch(operation);
-    });
+    worker.instance.match(worker.config.getPlatform(), worker.config.getRequeueOnFailure(), this::fetch);
```

Comment:
Oh, cool, I didn't know about this new :: syntax!

Reply:
It was new to me too (I'm really not a Java guy) I had some help from a ex-googler colleague who hopefully will be able to be an objective reviewer here eventually